app/holds.js

```diff
@@ -11,7 +11,12 @@ const getHolds = async () => {
 
 const getHoldCategories = async () => {
   const { payload } = await get('/payment-hold-categories')
-  const schemes = [...new Set(payload.paymentHoldCategories.map(item => item.schemeName))]
+  const schemes = [...new Set(payload.paymentHoldCategories.map((item) => {
+    if (item.schemeName === 'Vet Visits') {
+      item.schemeName = 'Annual Health and Welfare Review'
+    }
+    return item.schemeName
+  }))]
```

Comment:
I'd maybe consider pulling the function within `map` into it's own function below this one.

Just so it's a bit simpler for a reader to understand.

For example.

```
...
 const schemes = [...new Set(payload.paymentHoldCategories.map(mapScheme)]
...

const mapScheme = (scheme) => {
  (scheme) => {
      if (scheme.schemeName === 'Vet Visits') {
        scheme.schemeName = 'Annual Health and Welfare Review'
      }
      return scheme.schemeName
 })
}

```

Something like that?

Also I'd try to avoid using `item` as it's a bit generic.  Sometime unavoidable but always good to make it as easy  as possible for a reader to understand what the variable represents.  Especially in languages without types like JavaScript where it's all guess work anyway!

Reply:
Updated as suggested, except put the new fucntion above as sonarlint didnt like it being declared after it was called.

Did think I could have split it out but didnt want to have to change the name of variable passed through to nunjucks. Didnt think to pass the function into the map. Thanks for feedback.